portfolio/src/main/webapp/style.css

```diff
@@ -69,4 +70,43 @@ ul {
   font-family: "karla";
   border-radius: 4px;
   background-color: #779be7;
-}
\ No newline at end of file
+}
+
+.myDiv {
+  text-align: center;
+}
+
+.btn-group .button1 {
+  background-color: #9f9fed;
+  border: none;
+  color: white;
+  border: 5px #662e9b;
+  padding: 14px 25px;
+  /*border-radius: 4px;*/
```

Comment:
No need to change it here, but remember in your group work to remove commented-out code before sending out your reviews.